site/docs/2.2.0/rdd-programming-guide.html

```diff
@@ -1088,7 +1088,7 @@ <h3 id="transformations">Transformations</h3>
 </tr>
 <tr>
   <td> <b>sample</b>(<i>withReplacement</i>, <i>fraction</i>, <i>seed</i>) </td>
-  <td> Sample a fraction <i>fraction</i> of the data, with or without replacement, using a given random number generator seed. </td>
+  <td> Sample a <i>fraction</i> of the data, with or without replacement, using a given random number generator seed. </td>
```

Comment:
This isn't a typo. It's saying that the fraction is given by the argument _fraction_. 
Also, this is a PR against a copy of the docs generated for Spark 2.2.0. You would fix docs in the apache/spark source tree for the next release.

Reply:
args are specified in the left field. looks to be a typo 
![image](https://user-images.githubusercontent.com/48096234/189198770-f7c6f852-2307-415a-9d81-a9eb09c12af1.png)
 Also do we just correct this file then https://github.com/apache/spark/blob/master/docs/rdd-programming-guide.md even for the site?

Reply:
i was reading contribution gyuideline and it said to updatate here as well

Reply:
Yes you would update that .md file, but I do not think this is a typo.

Reply:
I felt that the italics was enough for giving away its an arg, repetition would not be right.